src/Storages/System/StorageSystemDisks.cpp

```diff
@@ -64,9 +64,9 @@ Pipe StorageSystemDisks::read(
     {
         col_name->insert(disk_name);
         col_path->insert(disk_ptr->getPath());
-        col_free->insert(disk_ptr->getAvailableSpace());
-        col_total->insert(disk_ptr->getTotalSpace());
-        col_unreserved->insert(disk_ptr->getUnreservedSpace());
+        col_free->insert(disk_ptr->getAvailableSpace().value_or(std::numeric_limits<UInt64>::max()));
```

Comment:
Here I keep the old behavior, although it might be better to change to Nullable and return NULL in subsequent modifications.